python/langsmith/_internal/_operations.py

```diff
@@ -256,18 +257,23 @@ def serialized_run_operation_to_multipart_parts_and_context(
                     " periods ('.'). Please rename the attachment and try again."
                 )
                 continue
-
-            acc_parts.append(
-                (
-                    f"attachment.{op.id}.{n}",
+            
+            if isinstance(valb, Path):
+                #TODO: actually deal with this case
+                # This is just for speed of getting something out
+                continue
+            else:
+                acc_parts.append(
                     (
-                        None,
-                        valb,
-                        content_type,
-                        {"Content-Length": str(len(valb))},
-                    ),
+                        f"attachment.{op.id}.{n}",
+                        (
+                            None,
+                            valb,
+                            content_type,
+                            {"Content-Length": str(len(valb))},
+                        ),
+                    )
```

Comment:
I believe this is the only other place we access the attachments, but I could definitely be missing some. I just skipped it in this case for speed of getting something out, but we should fix this quickly after